docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/FeatureSelection/SelectFeaturesBasedOnCount.cs

```diff
@@ -0,0 +1,117 @@
+            //    6. Single Epithelial Cell Size   1 - 10
+            //    7. Bare Nuclei                   1 - 10
+            //    8. Bland Chromatin               1 - 10
+            //    9. Normal Nucleoli               1 - 10
+            //   10. Mitoses                       1 - 10
+
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // First, we define the loader: specify the data columns and where to find them in the text file. Notice that we combine entries from
+            // all the feature columns into entries of a vector of a single column named "Features".
+            var loader = mlContext.Data.CreateTextLoader(
+                columns: new[]
+                    {
+                        new TextLoader.Column("Label", DataKind.Boolean, 0),
+                        new TextLoader.Column("GroupA", DataKind.Single, new [] { new TextLoader.Range(1, 3) }),
+                        new TextLoader.Column("GroupB", DataKind.Single, new [] { new TextLoader.Range(4, 6) }),
+                        new TextLoader.Column("GroupC", DataKind.Single, new [] { new TextLoader.Range(7, 9) }),
+                    },
+                hasHeader: true
+            );
+
+            // Then, we use the loader to load the data as an IDataView.
+            var data = loader.Load(dataFilePath);
+
+            Console.WriteLine("Contents of column 'GroupB'");
+            PrintDataColumn(data, "GroupB");
+            // 5 7 10
+            // 1 2 2
+            // 1 3 4
+            // 3 2 1
+            
+            // Second, we define the transformations that we apply on the data. Remember that an Estimator does not transform data
+            // directly, but it needs to be trained on data using .Fit(), and it will output a Transformer, which can transform data.
+
+            // In this example we define a CountFeatureSelectingEstimator, that selects slots in a feature vector that have more non-default 
+            // values than the specified count. This transformation can be used to remove slots with too many missing values.
+            var pipeline = mlContext.Transforms.FeatureSelection.SelectFeaturesBasedOnCount(
+                outputColumnName: "FeaturesSelectedGroupB", inputColumnName: "GroupB", count: 695);
+
+            // The pipeline can then be trained, using .Fit(), and the resulting transformer can be used to transform data. 
+            var transformedData = pipeline.Fit(data).Transform(data);
+
+            Console.WriteLine("Contents of column 'FeaturesSelectedGroupB'");
+            PrintDataColumn(transformedData, "FeaturesSelectedGroupB");
+            // Note, SelectFeaturesBasedOnCount retained only 2 slots (out of 3).  
+            // 5 7
+            // 1 2
+            // 1 3
+            // 3 2
```

Comment:
with current data it's not obvious at all. Can we switch to some small in memory sample rather than unknown dataset?

Reply:
true. moved over to use a small in-memory dataset instead

---
In reply to: [271912555](https://github.com/dotnet/machinelearning/pull/3184#discussion_r271912555) [](ancestors = 271912555)